fhir-model/src/test/java/com/ibm/fhir/model/spec/test/R4ExamplesDriver.java

```diff
@@ -180,56 +177,60 @@ public void processExample(String file, Expectation expectation) throws ExampleP
      * @throws ExampleProcessorException
      */
     public void submitExample(List<ExampleProcessorException> errors, String file, Format format, Expectation expectation) throws ExampleProcessorException {
-        if (pool != null) {
-            // Wait until we have capacity. We do this to throttle the number of requests
-            // submitted to pool, hopefully avoiding memory issues if ever we have a really
-            // large index to process
-            lock.lock();
-            while (this.currentlySubmittedCount == this.maxInflight) {
-                try {
-                    this.inflightCondition.await(1000, TimeUnit.MILLISECONDS);
-                }
-                catch (InterruptedException x) {
-                    // NOP
-                }
+        if (pool == null) {
```

Comment:
why the switch? Now there are multiple exits from the method.

Reply:
just a preference thing.  with the long if block i find it hard to remember the context when we get to that bottom else.  i can change it back to how it was or i could just handle this case first, remove the return, and put the rest into an else